src/cljs/velho_ds/organisms/heading.cljs

```diff
@@ -92,3 +115,45 @@
        (into [:div (stylefy/use-style (merge style/page-heading-container
                                              {:display (if @sub-content-open? "block" "none")}))]
              (for [item sub-content] ^{:key item} item))])))
+
+(defn content-header [{:keys [styles] :as options} & content]
+  (if (map? options)
+    [grid/grid-wrap {:rows 1, :cols (count content), :styles styles}
+     (for [item content]
+       ^{:key item} [grid/grid-cell {:styles {:display "grid"}} item])]
+    [grid/grid-wrap {:rows 1, :cols (+ 1 (count content))}
+     [grid/grid-cell {:styles {:display "grid"}} options]
+     (for [item content]
+       ^{:key item} [grid/grid-cell {:styles {:display "grid"}} item])]))
+
+(defn content-header-default [{:keys [breadcrumb
+                                      footnote
+                                      meta
+                                      navigation
+                                      theme-color
+                                      info-icon
+                                      info-keyvalues
+                                      styles]}]
+  [content-header (tools-style/merge-styles styles {:background color/color-white
+                                                    :grid-template-columns "1fr minmax(12rem, 25%)"})
+   [content-info {:breadcrumb breadcrumb
+                  :footnote footnote
+                  :meta (for [button meta]
+                          ^{:key button} [buttons/primary-small {:content (:content button)
+                                                                 :icon (:icon button)
+                                                                 :on-click-fn (:on-click-fn button)}])
+                  :navigation (for [nav navigation]
+                                ^{:key nav} [buttons/icon-link {:icon (:icon nav)
+                                                                :label (:label nav)
+                                                                :active (:active nav)
+                                                                :on-click-fn (:on-click-fn nav)}])
+                  :bar-color theme-color}]
```

Comment:
Should there be some default theme-color in case no theme-color is given?

Reply:
good catch, set it to neutral-4